indico/modules/events/logs/client/js/components/EventLog.jsx

```diff
@@ -6,13 +6,45 @@
 // LICENSE file for more details.
 
 import React from 'react';
+import {useDispatch, useSelector} from 'react-redux';
+
+import {Param, Translate} from 'indico/react/i18n';
 
 import LogEntryList from '../containers/LogEntryList';
 import Toolbar from './Toolbar';
+import {fetchLogEntries, setDataQuery, setPage} from '../actions';
+
+function DataQueryMessage() {
+  const dispatch = useDispatch();
+  const dataQuery = useSelector(state => state.logs.dataQuery);
+
+  function resetUrlFiltering() {
+    dispatch(setDataQuery({}));
+    dispatch(setPage(1));
+    dispatch(fetchLogEntries());
+  }
+
+  if (Object.keys(dataQuery).length === 0) {
+    return null;
+  }
+  return (
+    <>
+      <div className="highlight-message-box">
+        <span className="icon"></span>
+        <div className="message-text">
+          <Translate>
+            Some entries may currently be filtered by URL. <Param name="link" wrapper={<a onClick={resetUrlFiltering} />}>Click here</Param> to disable URL filtering.
```

Comment:
I don't like the term "url filtering" - it's not really clear what that's supposed to mean! Can we find a better word for it?

> Log entries are currently filtered. Click hear to [clear the filter].

What about something like this? Even though it might be a bit misleading because there's this filter and the search box... which you wouldn't want to clear by clicking here.

Reply:
> Log entries **are** currently filtered. Click here to [clear the filter].

We don't know for sure if entries are filtered in the current implementation. I would still keep __"may be"__.

> Even though it might be a bit misleading because there's this filter and the search box... which you wouldn't want to clear by clicking here.

Exactly. I would rather make a clear distinction between the two kinds of filtering. How about something like this?

> Log entries may be currently filtered by URL. Click here to clear the filter.

"Filtered by URL" is more semantically accurate than "URL filtering" and, although tech-savvy, when looking at the URL (`?filter.xyz=...`) one can imagine what is going on.

Reply:
I think they are always being filtered, even if the filter happens to match everything. But that is actually impossible because "event created" will never show up with such a filter active.